### ISSUE TITLE
SSR is not supported in Solid

### ISSUE THREAD

**Issue by paularmstrong** [inokawa/virtua#517]
**Describe the bug**

Server side rendering crashes Solid Start server with the following syntax error:

```
SyntaxError: The requested module 'solid-js/web' does not provide an export named 'use'
```

**To Reproduce**

Add and render any of `VList`, `WindowVirtualizer`, etc in a solid-start application with server-side rendering enabled.

**Expected behavior**

The README for virta says that SSR is supported across the board.

**Platform:**

- OS: n/a
- Browser: n/a
- Version of this package: 0.34.2
- Version of framework:
  - `solid-js@1.8.22`
  - `@solidjs/start@1.0.6`
  - 
**Additional context**

I noticed there are SSR tests for other frameworks like React and Vue, but none for Solid in this repo.

**Comment by inokawa** [comment]
https://github.com/chakra-ui/ark/issues/1363

**Comment by inokawa** [comment]
Probably we need to publish .jsx in `solid` export condition like .svelte for Svelte.
https://github.com/chakra-ui/ark/blob/02d0840723b746d5dbb0223b56625fb8352a33f3/packages/solid/package.json#L62
https://www.solidjs.com/guides/server#server-side-rendering

**Comment by inokawa** [comment]
Fixed in `0.38.2`.
